Makefile

```diff
@@ -418,6 +418,10 @@ npm-check:
 		exit 1; \
 	fi;
 
+.PHONY: webpack
+webpack: npm
+	npm install webpack -g
+
```

Comment:
This is unnecessary, just run `npm install --save-dev webpack` which will add it as a proper dependency.